extensions/wikia/AdEngine/js/template/interstitial.js

```diff
@@ -0,0 +1,60 @@
+/*global define*/
+define('ext.wikia.adEngine.template.interstitial', [
+	'wikia.document',
+	'wikia.log',
+	'wikia.ui.factory',
+	'wikia.window'
+], function (doc, log, uiFactory, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.template.interstitial',
+		modalId = 'ext-wikia-adEngine-template-interstitial',
+		modalPadding = 20,
+		pollingInterval = 1000; // 1s
+
+	function show(params) {
```

Comment:
It would be nice to include a doc block with some of the required properties of `params`, like `slot` and `elem`.